### PR TITLE
chore: add funding field to package.json

### PR DESCRIPTION
**Pull request by armano2**
This PR removes `opencollective-postinstall` that was used only during dev build and adds a `funding` field to the `package.json` so users installing will know how to contribute.

This is a new feature as of `npm` version 6.13.

See:
- https://medium.com/npm-inc/npm-weekly-222-announcing-lots-of-launches-2de4bd670340
- https://github.com/npm/rfcs/blob/latest/accepted/0017-add-funding-support.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "lint-fix": "eslint . --ext .js,.ts --fix",
     "pre-commit": "yarn lint-staged",
     "pre-push": "yarn format-check",
-    "postinstall": "lerna bootstrap && yarn build && lerna link && npm run check-clean-workspace-after-install && opencollective-postinstall",
+    "postinstall": "lerna bootstrap && yarn build && lerna link && npm run check-clean-workspace-after-install",
```
Comment (bradzacher) [inline package.json]: question:
`npm run check-clean-workspace-after-install` is pretty annoying here.
The intention of it is to ensure there aren't any lockfile changes in the CI

Do we want to move this to the azure script?

Comment (armano2) [reply package.json]: i don't like it to, its fails always when you have local changes

**Comment by armano2** [comment]
> probably also want to add it to every single one of our packages.

there is actually no need for that as every person who installs one of those packages has must have ts-estree or plugin installed

**Comment by armano2** [comment]
> probably also want to add it to every single one of our packages.

i added it to all packages except tslint as this is only wrapper

**Pull request merged**